src/bedrock/route/seed_handler.cpp

```diff
@@ -37,5 +36,5 @@ void seed_handler(
 
   std::string serialized;
   membership.SerializeToString(&serialized);
-  zmq_util::send_string(serialized, addr_responder);
+  return serialized;
```

Comment:
Why don't we just call `kZmqUtilInterface->send_string` here?

Reply:
because we decided to not include the zmq socket in the handler

Reply:
Oh, right. 🤦‍♂️ Sorry.